k8s/cmd/commands/flags/sidecar_flags.go

```diff
@@ -22,9 +22,35 @@ import (
 )
 
 var (
+	// the following label is for vineyard rpc service
+	// DefaultSidecarLabel is the default name of the vineyard sidecar container
+	DefaultSidecarLabel = "sidecar.v6d.io/name"
+
+	// DefaultSidecarName = "vineyard-sidecar"
+	DefaultSidecarName = "vineyard-sidecar"
+
+	// SidecarName is the name of sidecar
+	// it is also the label selector value of sidecar
+	SidecarName string
+
 	// WorkloadYaml is the yaml of workload
 	WorkloadYaml string
 
+	// WorkloadResource is the resource of workload
+	WorkloadResource string
+
+	// DeployExternalEtcd is the flag of whether to deploy external etcd
+	DeployExternalEtcd bool
+
+	// DeployRPCService is the flag of whether to deploy service for vineyard cluster
+	DeployRPCService bool
+
+	// DeployEtcdService is the flag of whether to deploy service for etcd cluster
+	DeployEtcdService bool
+
+	// OutputFormat is the output format of the command
+	OutputFormat string
```

Comment:
DeployExternalEtcd -> UseExternalEtcd
DeployRPCService -> EnableRPCService
DeployEtcdService: how you can deploy the etcd without enabling its service?

Reply:
> DeployEtcdService: how you can deploy the etcd without enabling its service?

It's to support GraphScope as the coordinator will create an etcd service.